packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -957,13 +938,15 @@ export const addSandboxesToFolder: AsyncAction<{
 
 export const createTeam: AsyncAction<{
   teamName: string;
-}> = async ({ effects, actions, state }, { teamName }) => {
+  pilot?: boolean;
+}> = async ({ effects, actions, state }, { teamName, pilot }) => {
```

Comment:
No need for double negation if you pass a default value

```suggestion
}> = async ({ effects, actions, state }, { teamName, pilot = false }) => {
```